server/v2/stf/stf.go

```diff
@@ -379,11 +383,11 @@ func (s STF[T]) clone() STF[T] {
 	}
 }
 
-// executionContext is a struct that holds the context for the execution of a tx.
-type executionContext struct {
+// ExecutionContext is a struct that holds the context for the execution of a tx.
+type ExecutionContext struct {
```

Comment:
Is it made public only for testing purposes?
I thought we shouldn't leak stf internal context and only interact with it thanks to services.
Should we not create a file called `export_test.go` and make an alias there that we use in tests?

Reply:
Yeah it makes sense, just for testing purpose